client/src/deleteButton.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import '../style.sass';
+
+export class DeleteButton extends React.Component {
```

Comment:
Why does this need its own class over defining `deleteSighting` in a `sightingAsText` class (moved out from `index.js`?

Do we get some benefit from it?

Reply:
This will become button once that PR approved

Reply:
Right now I've opted to keep the DeleteButton component - the big thing being that the ID of the sighting is passed to it, but not to the button.

I guess we could move sightingAsText to a component (currently function), with the function and ID being part of that, then just passing the function to the button, which would no longer need the ID (since it's part of the function already)

Reply:
No longer use `deleteButton`, instead have delete function inside `sightingAsText` and pass function to `button` class